.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Captum Deploy Website
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+
+  workflow_dispatch:
+
+env:
+  CHANNEL: "nightly"
+
+jobs:
+  tests:
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.12xlarge
+      docker-image: cimg/python:3.6-node
+      repository: pytorch/captum
+      script: |
+        sudo chmod -R 777 .
+        ./scripts/install_via_pip.sh -n -d
+
+        if [[ $CIRCLE_PROJECT_USERNAME == "pytorch" && -z $CI_PULL_REQUEST && -z $CIRCLE_PR_USERNAME ]]; then
```

Comment:
`CIRCLE_PROJECT_USERNAME` does not have a set value. Any env vars without a set value with fail with `unbound environment variable` since we runs this shell script with `set -u`.